compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/AggregatedOpInterfaceImpl.cpp

```diff
@@ -322,6 +357,14 @@ OnlineAttentionOp::decomposeOperation(OpBuilder &b) {
                                                offset);
   }
 
+  // S += mask
+  if (*mask) {
+      s = applyMask(b, loc, sMap, *getMaskMap(), s, *mask);
+  }
+
+  Value log2e = b.create<arith::ConstantOp>(loc, b.getFloatAttr(scale.getType(), M_LOG2E));
+  s = elementwiseValueInPlace<arith::MulFOp>(b, loc, sMap, scaleMap, s, log2e);
```

Comment:
Hmm in here we are doing scaling on a tensor with BxMxK2 instead of BxMxK1. where K2 is seq_len and K1 is hidden_dim. For SDXL perf should be OK, but for Llama where seq_len/K2 can get really really large, it may make perf slower. 

We can do perhaps do some if regions to do scaling after mm1 iff mask is not i1/boolean mask. Alternatively, we can also omit the support for non boolean mask since I don't think we use it much and this way can keep the code clean.